.github/workflows/release.yaml

```diff
@@ -167,14 +139,14 @@ jobs:
           cd dist/
           zip -r composio-linux-amd64.zip bin/*
           rm -rf bin/
```

Comment:
The macOS version used in the CLI build matrix has changed from `macos-12` to `macos-13`. While `macos-13` is likely a more up-to-date and supported version, this change might introduce compatibility issues with older macOS versions.  Consider adding a check to ensure compatibility or reverting to `macos-12` if backward compatibility is critical.